libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/preview/CarRouteLine.kt

```diff
@@ -100,7 +107,7 @@ class CarRouteLine internal constructor(
         val locationPlugin = mapboxCarMapSurface.mapSurface.location
         styleLoadedListener = mapboxCarMapSurface.handleStyleOnAttached { style ->
             val routeLineOptions = getMapboxRouteLineOptions(style)
-            routeLineView = MapboxRouteLineView(routeLineOptions)
+            val routeLineView = MapboxRouteLineView(routeLineOptions).also { routeLineView = it }
```

Comment:
nit: we don't need the `.also { assign }` here because the values are `private lateinit var`

``` kotlin
routeLineView = MapboxRouteLineView(routeLineOptions)
routeLineApi = MapboxRouteLineApi(routeLineOptions)
routeLineView.initializeLayers(style)
```